components/search/MoreFacetsDropdownOption.vue

```diff
@@ -0,0 +1,36 @@
+<template>
```

Comment:
Is this component used any more? If not, delete it.

Reply:
Now removed